### DISCUSSION TITLE
__VUE_PROD_DEVTOOLS__ is not defined

### DISCUSSION THREAD

**Discussion by wulucxy** [vuejs/pinia#1567] [Help and Questions] [answered]
### Reproduction

after upgrade to version 2.0.19 this happens

### Steps to reproduce the bug

after version 2.0.19 this error happens

### Expected behavior

not error

### Actual behavior

version: 2.0.19

**Comment by youcanping** [comment]
今天更新我的项目也起不来了，目前是锁定 2.0.18

**Comment by posva** [comment, marked as answer]
do you have a boiled down repro for this? Vue also has that global defined and it's replaced by the bundler: https://github.com/vuejs/core/tree/main/packages/vue#bundler-build-feature-flags

It's weird it doesn't fail with Vue or Vue router as they also have it:

- https://unpkg.com/browse/@vue/runtime-core@3.2.37/dist/runtime-core.esm-bundler.js
- https://unpkg.com/browse/vue-router@4.1.3/dist/vue-router.mjs
- https://unpkg.com/browse/pinia@2.0.19/dist/pinia.mjs

Here are some workarounds: https://github.com/vuejs/pinia/issues/1570#issuecomment-1220632205

**Reply by posva** [reply]
aah, I see where the problem comes from. This is Nuxt 2 only

**Reply by posva** [reply]
This should be fixed in the latest release BTW

**Reply by fMeow** [reply]
I don't think it is nuxt only.  @posva 

Came across the same issue here, vue 2 with vue-cli in prod mode. 
<img width="308" alt="image" src="https://user-images.githubusercontent.com/16273287/185822779-6d192b87-b8c5-46d5-bd69-82fd47024a1e.png">

**Reply by posva** [reply]
it's Vue 2 + webpack. It's solved now anyway

**Comment by darthf1** [comment]
Same issue here, when upgrading from `2.0.18` to `2.0.19`. It only happens on production mode. 

![image](https://user-images.githubusercontent.com/17253332/185606712-eaeb7415-ad2a-46ba-afd9-9acc4d2194ff.png)
